docs/components/skeleton.md

```diff
@@ -0,0 +1,145 @@
+# Skeleton / Loading State
+
+**KSkeleton** is used as a placeholder component while the primary content is still loading.
+
+## Props
+### type
+- `type`
+
+There are 5 different types of loading states that KSkeleton supports: Card, Table, Form, Spinner and a generic loading state. Defaults to a generic loading state. The following example shows a Form type KSKeleton.
+
+<template>
+  <KSkeleton type="form" />
+</template>
+
+```vue
+<template>
+  <KSkeleton type="form" />
+</template>
+```
+
+### delayMilliseconds
+- `delay-milliseconds`
+
+The number of milliseconds to show this skeleton state. Defaults to 750. The following example shows a delay of 1 second.
+
+<template>
+  <KSkeleton type="form" delay-milliseconds="1000" />
```

Comment:
this isn't going to work because you need to add it to the component register https://github.com/Kong/kongponents/blob/master/docs/.vuepress/enhanceApp.js